client/src/components/Libraries/LibraryEditField.vue

```diff
@@ -3,7 +3,7 @@
         <div class="text-field">
             <!-- edit mode -->
             <div v-if="isEditMode">
-                <textarea class="form-control" v-model="textField" rows="3" />
+                <b-form-textarea class="form-control" :value="text" @change="updateValue" rows="3" no-resize />
```

Comment:
did v-model not work here ?

Reply:
Oh I see, in `updateValue` you probably want to emit the new value back to the parent, where the parent component should fire a request and update the value that gets passed in as a prop

Reply:
When we change the page, the `text` prop is getting updated. 
However `textField` is assigned to text only on creation https://github.com/galaxyproject/galaxy/blob/dev/client/src/components/Libraries/LibraryEditField.vue#L58
so I made `:value` to be equal prop and change `textField` only on `change` event

Reply:
> Oh I see, in `updateValue` you probably want to emit the new value back to the parent, where the parent component should fire a request and update the value that gets passed in as a prop

:+1:

Reply:
Are you going to do this then ? That means you can replace https://github.com/mvdbeek/galaxy/blob/06819532f1a317ee91aed1936c788ee79f911aba/client/src/components/Libraries/LibrariesList.vue#L248-L249. I also noticed that the name field is editable, but it has no handler attached.